tests/conftest.py

```diff
@@ -36,23 +37,44 @@ def monkeysession():
         yield mp
 
 
+@pytest.fixture()
+def lora_adapter_name():
+    return "lora-test"
+
+
+@pytest.fixture()
+def lora_adapter_path():
+    from huggingface_hub import snapshot_download
+
+    path = snapshot_download(repo_id="yard1/llama-2-7b-sql-lora-test")
+    return path
```

Comment:
This is the same adapter used in the docs: https://docs.vllm.ai/en/latest/models/lora.html